libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -149,9 +149,18 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("INPUT_TC", 20, AC_AttitudeControl, _input_tc, AC_ATTITUDE_CONTROL_INPUT_TC_DEFAULT),
 
+    // @Param: THR_G_BOOST
+    // @DisplayName: Throttle-gain boost
+    // @Description: Throttle-gain boost ratio. A value of 0 means no boosting is applied, a value of 0 means full boosting is applied. Describes the ratio increase that is applied to angle P and PD on pitch and roll.
+    // @Range: 0 1
+    // @User: Advanced
+    AP_GROUPINFO("THR_G_BOOST", 21, AC_AttitudeControl, _throttle_gain_boost, 0.0f),
+
     AP_GROUPEND
 };
 
+constexpr Vector3f AC_AttitudeControl::UNIT_VECTORF;
```

Comment:
Is it weird to have this here in the middle?  I would have expected it at the top of the file.

Reply:
Its at the top